cpp/cmake/cudf-config.cmake.in

```diff
@@ -83,6 +83,9 @@ find_dependency(ArrowCUDA @CUDF_VERSION_Arrow@)
 
 find_dependency(rmm @CUDF_MIN_VERSION_rmm@)
 
+find_dependency(nvCOMP @CUDF_MIN_VERSION_nvCOMP@)
```

Comment:
Drop the `find_dependency` line, it isn't needed as the target is created by `include("${CMAKE_CURRENT_LIST_DIR}/cudf-nvcomp-target.cmake")`